js/templates/settings.html

```diff
@@ -0,0 +1,463 @@
+<div class="mainContainer marginTop50">
+
+<div class="topThumb"
+     style="
+     background-image: url(<%- ob.user.server_url %>get_image?hash=<%- ob.page.profile.avatar_hash %>);">
```

Comment:
After merging with the latest master, take a look at the avatar in the user page template. We're now adding a second background image (the default image) to the avatars, it would be good to update this to do the same.